index.js

```diff
@@ -28,28 +31,46 @@ module.exports = {
     "plugin:prettier/recommended",
     "plugin:sonarjs/recommended",
     "plugin:typescript-sort-keys/recommended",
-    "plugin:unicorn/recommended",
   ],
   parserOptions: {
-    ecmaVersion: "latest",
+    ecmaVersion: 2022,
     project: "./tsconfig.json",
-    sourceType: "module",
   },
+  overrides: [
+    {
+      files: ["test/**"],
+      plugins: ["jest"],
+      rules: {
+        // this turns the original rule off *only* for test files, for jest compatibility
+        "@typescript-eslint/unbound-method": "off",
+        "jest/unbound-method": "error",
+      },
+    },
+  ],
   rules: {
     "import/default": "off",
     "import/named": "off",
     "import/namespace": "off",
     "import/no-default-export": "error",
     "import/no-extraneous-dependencies": [
       "error",
-      { devDependencies: ["src/**/__tests__/**"] },
+      {
+        devDependencies: ["test/**/*.tsx", "test/**/*.ts"],
+      },
     ],
+    "import/export": "off",
     "import/prefer-default-export": "off",
     "jest/no-jest-import": "off",
     "node/no-unpublished-import": [
       "error",
       {
-        allowModules: ["@jest/globals"],
+        allowModules: [
+          "@jest/globals",
+          "@testing-library/dom",
+          "@testing-library/react",
+          "@testing-library/user-event",
+          "nock",
```

Comment:
Removed here: https://github.com/open-turo/eslint-config-typescript/pull/118/commits/5aa24f91e46f3f2717aea032e4c97357e756228f
And added here: https://github.com/open-turo/eslint-config-react/pull/85. Since this `react` repo is very tied to our front-end repositories config, it felt right to keep it there